hypha/home/models.py

```diff
@@ -1,28 +1,7 @@
-from django.db import models
-from wagtail.admin.panels import FieldPanel
 from wagtail.models import Page
-from wagtail.search import index
-
-from hypha.apply.funds.models import ApplicationBase, LabBase
 
 
 class ApplyHomePage(Page):
     # Only allow creating HomePages at the root level
     parent_page_types = ["wagtailcore.Page"]
     subpage_types = ["funds.FundType", "funds.LabType", "funds.RequestForPartners"]
-
-    strapline = models.CharField(blank=True, max_length=255)
```

Comment:
Removing this line makes the code in `copy_homepage_title_and_strapline` not run since the model no longer have a strapline.

Would be really good to get this working making the deployment of this release seamless.

Maybe we can remove the strapline fileld in a separate release?